README.md

```diff
@@ -1,2 +1,50 @@
 # system_uri
+
 Desktop System App URI registration handler 
+
+**Maintainer:** Spandan Sharma (spandan.sharma@maidsafe.net)
+
+|Crate|Linux/OS X|Windows|Coverage|Issues|
+|:---:|:--------:|:-----:|:------:|:----:|
+|[![](http://meritbadge.herokuapp.com/system_uri)](https://crates.io/crates/system_uri)|[![Build Status](https://travis-ci.org/maidsafe/system_uri.svg?branch=master)](https://travis-ci.org/maidsafe/system_uri)|[![Build status](https://ci.appveyor.com/api/projects/status/c61jthx04us5j57j/branch/master?svg=true)](https://ci.appveyor.com/project/MaidSafe-QA/system_uri/branch/master)|[![Coverage Status](https://coveralls.io/repos/maidsafe/system_uri/badge.svg?branch=master)](https://coveralls.io/r/maidsafe/system_uri?branch=master)|[![Stories in Ready](https://badge.waffle.io/maidsafe/system_uri.png?label=ready&title=Ready)](https://waffle.io/maidsafe/system_uri)|
```

Comment:
This is the 3rd time - you are not follwing the stuff mentioned in the review: https://github.com/maidsafe/system_uri/pull/3#pullrequestreview-12488086
and
https://github.com/maidsafe/system_uri/pull/3#discussion_r92156323